### ISSUE TITLE
Implementing inherit/initial/unset properties

### ISSUE THREAD

**Issue by elidickinson** [peterbe/premailer#106]
Hello,

I'm using premailer to inline some templates that are getting rather large (I'm using the Zurb Ink framework).

It would be very helpful to be able to "[unset](https://developer.mozilla.org/en-US/docs/Web/CSS/unset)" certain properties so that they aren't inlined. I think this should be easy to implement since, as far as I can tell, `inherit`, `initial`, and `unset` don't really mean anything when written to a property in a `style=` attribute. So I think those could properties could simply be omitted and it would have the effect of reverting them to their inherited or initial value.

**Comment by peterbe** [comment]
I can't even find `unset` [on caniuse.com](http://caniuse.com/#cats=CSS)
And according to MDN the support is really quite poor. How does that matter? 
I'm not very familiar with this feature.

**Comment by elidickinson** [comment]
@peterbe `unset` is in Chrome since 41 and Firefox since 27, but maybe that's beside the point. Let's look at `inherit` instead, which is pretty widely supported.

Imagine I've got CSS like this
```
body, p, td { 
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
}
```

But I would prefer NOT to have that style apply to my spacer table cells (they have no text in them anyway) so I add:

```
td.spacer {
    font-family: inherit;
}
```

That works fine, but it actually writes `font-family: inherit` into the `style=` attribute... I'd much prefer it to simply omit `font-family` (which I *think* is always functionally equivalent -- it will simply inherit from the parent "by default" absent any specific directive).

Admittedly it's a bit of an odd request, but I'm running into hard filesize limits with my emails after inlining, so I either need a way to direct premailer to ignore certain rules or I need to write a post-processor.

**Comment by peterbe** [comment]
Do you have a lot of these `inherit` things so that it would be worth some savings?
I can definitely see a significant bytes savings on not having to have `font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;` on EVERY `p` or `td` element. 

With premailer we can always add more options, if people need them, and make them switched off by default so that it doesn't have a too scary impact on existing users. 
In other words, let's write a patch.

**Comment by elidickinson** [comment]
Greetings, I have finally gone ahead and written a patch to address this: #136

I have decided to use `unset` only (and not `inherit` or `initial`) because I think there are some edge cases where inlining "font-size:inherit" is different from leaving it out of the `style=""` completely. Unset should always have exactly the same meaning, based on my reading of the CSS 3 spec.
